index.js

```diff
@@ -15,7 +15,7 @@ app.set('views', path.join(__dirname, 'src/views'))
 app.set('view engine', 'html')
 app.use(express.static(path.join(__dirname, 'src/public')))
 
-app.get('/', async (req, res) => {
+app.get('/', (req, res) => {
   tallyForAllChannels(token, message_limit).then(mojis => {
```

Comment:
Might be good to handle the error case here, even if it's unlikely - the other `.then` callback

Reply:
Definitely, I wish it were possible to lint for promises that don't handle the error case, I sometimes forget it 😬.